pepper-apis/src/main/java/org/broadinstitute/ddp/analytics/GoogleAnalyticsMetrics.java

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.ddp.analytics;
+
+public class GoogleAnalyticsMetrics {
+
+    public static final String EVENT_CATEGORY_USER_REGISTRATION = "user-registration";
+    public static final String EVENT_ACTION_USER_REGISTRATION = "register-user";
+    public static final String EVENT_LABEL_USER_REGISTRATION = "registration"; //studyGuid prepended
+
+    public static final String EVENT_CATEGORY_USER_LOGIN = "user-login";
+    public static final String EVENT_ACTION_USER_LOGIN = "user-logged-in";
+    public static final String EVENT_LABEL_USER_LOGIN = "login"; //studyGuid prepended
+
+    public static final String EVENT_CATEGORY_ACTIVITY_INSTANCE = "activity";
+    public static final String EVENT_ACTION_ACTIVITY_INSTANCE = "activity-instance";
+    public static final String EVENT_LABEL_ACTIVITY_INSTANCE = "get-activity-instance"; //studyGuid & activityCode prepended
```

Comment:
What's the advantage of prepending? Does it make it easier to search or make it more organized in the UI/web console? Have we considered appending instead?

* `registration:ANGIO`
* `login:cmi-mbc`
* `get-activity-instance:testboston:CONSENT`